include/modem/location.h

```diff
@@ -379,9 +442,24 @@ int location_agps_data_process(const char *buf, size_t buf_len);
  *
  * @return 0 on success, or negative error code on failure.
  * @retval -EINVAL Given buffer is NULL or buffer length zero.
+ * @retval -ENOTSUP CONFIG_LOCATION_GNSS_PGPS_EXTERNAL is not set.
  */
 int location_pgps_data_process(const char *buf, size_t buf_len);
 
+/**
+ * @brief Pass cellular location back to library.
+ *
+ * @details If Location library is not receiving cellular position directly from services,
+ * it triggers the LOCATION_EVT_CELLULAR_REQUEST event when neighbor cell information
+ * should be sent to cloud services for location resolution. Once application has obtained
+ * the location it can be handed over to Location library using this function.
+ *
+ * @param[in] location Cellular location results. NULL indicates that location
+ *                     couldn't be retrieved.
+ * @param[in] undefined Valid if location parameter is NULL. Defines whether
+ *                      cellular positioning result is unknown or failure.
```

Comment:
"failure" does not sound very good. Maybe something like "cannot be resolved".

Reply:
This sentence is not very good. I changed to but doesn't sounds perfect either:
`True if cellular positioning failed or the result is unknown. Otherwise false.`